appveyor.yml

```diff
@@ -0,0 +1,72 @@
+# This file was automatically generated by conda-smithy. To update a component of this
+# file, make changes to conda-forge.yml and/or recipe/meta.yaml, and run
+# "conda smithy rerender".
+
+environment:
+
+  # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+  # /E:ON and /V:ON options are not enabled in the batch script intepreter
+  # See: http://stackoverflow.com/a/13751649/163740
+  CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
+
+  BINSTAR_TOKEN:
+    # The BINSTAR_TOKEN secure variable. This is defined canonically in conda-forge.yml.
+    secure: KDvUkPrcdKmV2MxakkyQlrLA0rE/6WSuXxD42UezzZjCB0TRqnnOeebSW03/SnFP
+
+  matrix:
+    - TARGET_ARCH: x86
+      CONDA_PY: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 27
```

Comment:
Is there any particular reason for using python  2.7 here? It might be easier to start with 3.5 since this uses a much newer version of visual studio.